HdmiInput/HdmiInput.cpp

```diff
@@ -42,11 +42,14 @@
 #define HDMIINPUT_METHOD_START_HDMI_INPUT "startHdmiInput"
 #define HDMIINPUT_METHOD_STOP_HDMI_INPUT "stopHdmiInput"
 #define HDMIINPUT_METHOD_SCALE_HDMI_INPUT "setVideoRectangle"
+#define HDMIINPUT_METHOD_SUPPORTED_GAME_FEATURES "getSupportedGameFeatures"
+#define HDMIINPUT_METHOD_ALLM_STATUS "getHdmiALLMStatus"
```

Comment:
If ALLM is one of the same Feature and if you have multiple game feature should not this APIO should be generic to take Game mode as input param and return its status?